### PR TITLE
RHCLOUD-22647 update event-log filters to show correct status

### PR DESCRIPTION
**Pull request by aferd**
Before - 
Screen Shot 2022-11-17 at 10 44 14 AM

After - 
Screen Shot 2023-08-24 at 1 54 07 PM

### CODE REVIEW

File: src/components/Notifications/EventLog/EventLogToolbar.tsx [hunk 1]
```diff
@@ -61,6 +61,22 @@ const actionStatusMetadata = [
...
     },
+    {
+        value: 'false',
+        chipValue: 'Sent',
+        label: <span><CheckCircleIcon color='green' /> Sent</span>
+    },
+    {
+        value: 'false',
+        chipValue: 'Warning',
+        label: <span><ExclamationTriangleIcon color='#F0AB00' /> Warning </span>
```
Comment (karelhala) [inline src/components/Notifications/EventLog/EventLogToolbar.tsx]: Might be good if we could use CSS var `--pf-chart-donut--threshold--warning--Color` and custom class I am just not sure if it's applicable to icons.

**Review by karelhala — APPROVED** [review]
looking good! Just one nitpick about inline color.

File: src/components/Notifications/EventLog/EventLogToolbar.tsx [hunk 2]
```diff
@@ -55,12 +57,32 @@ const notificationTypes: Record<NotificationType, { name: string }> = {
     }
 };
 
+const warningIconColorClassName = style({
```
Comment (aferd) [inline src/components/Notifications/EventLog/EventLogToolbar.tsx:60]: like this? I found the global_color var for that color in PF.

Comment (karelhala) [reply src/components/Notifications/EventLog/EventLogToolbar.tsx:60]: Ah, we are using `typestyle` for styling. Yeah, this is the correct way!

**Review by karelhala — APPROVED** [review]

**Pull request merged**